spec/features/admin/admin_deletes_user_spec.rb

```diff
@@ -15,7 +15,7 @@
     expect(page).
       to have_text(I18n.t(
         "flash.actions.destroy.notice",
-        resource_name: "User"
-      ))
+        resource_name: "User",
+      ),)
```

Comment:
Suggestion to change the format to be:
```
    notice = I18n.t("flash.actions.destroy.notice", resource_name: "User")
    expect(page).to have_text(notice)
```

Reply:
Done in PR ad-refactor-flash-format-specs sha: 4be77c8